core_backend/app/auth/routers.py

```diff
@@ -2,6 +2,7 @@
 from fastapi.security import OAuth2PasswordRequestForm
 from google.auth.transport import requests
 from google.oauth2 import id_token
+from starlette.requests import Request
```

Comment:
\nit Probably the same but mind using?
```
from fastapi.requests import Request
```

Reply:
I am not sure if it's the same but I used it because it is the same used in `core_backend.app.prometheus_middleware.py`should I also change it there ?